README.md

```diff
@@ -1,204 +1,176 @@
 
+```zsh
+$ cd daml-projects/docs.daml.com
+direnv: loading ~/daml-projects/docs.daml.com/.envrc
+direnv: using nix
+direnv: export +AR +AR_FOR_TARGET +ARTIFACTORY_USERNAME +ARTIFACTORY_PASSWORD ...
+```
 
-## Making a release
+The bottom list is longer, but it is important that you see `ARTIFACTORY_USERNAME` and `ARTIFACTORY_PASSWORD`.
 
-Changes to `main` are immediately reflected on the live (versioned) website.
-The `LATEST` file contains a `prefix` entry, and the result of building `main`
-is automatically pushed under that prefix.
+:warning: The first time you do this, you won't see these variables, as you won't have had added them yet.
 
-For example, if `LATEST` looks like:
+:warning: Don't be tempted to give up when `direnv` tells you it's taking a while. Be patient.
 
-```json
-{
-  "daml": "2.1.0-snapshot.20220311.9514.0.ab6e085f",
-  "canton": "20220315",
-  "prefix": "2.1.0"
-}
-```
+### Environment variables
 
-then the result of building the documentation based on SDK
-`2.1.0-snapshot.20220311.9514.0.ab6e085f` and Canton `20220315` will be pushed
-to https://docs.daml.com/2.1.0, and should be live "immediately" (the build &
-push process takes about 10 minutes).
+Create or edit the `.envrc.private` file in the root directory to contain the following:
 
-Note that at this point the release process pushes the new files, overwriting
-existing ones, but does not at the moment delete anything (i.e. if a file no
-longer exists in a new version, there will still be a copy of the old version
-of that file).
+```plaintext
+export ARTIFACTORY_USERNAME=firstname.lastname
+export ARTIFACTORY_PASSWORD=Long_string_of_gibberish_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV
+```
 
-## Building locally
+- `ARTIFACTORY_USERNAME`: 
+  Find your username in [JFrog](https://digitalasset.jfrog.io/ui/admin/artifactory/user_profile); usually `firstname.lastname`.
+- `ARTIFACTORY_PASSWORD`: 
+  This is your **API Key**, *NOT* your password. Find your API key at the bottom of **Authentication Settings** in your profile.
 
-When working on the documentation, you'll want to see the results of your work
-locally before opening a pull request.
+## Build and view the docs locally
 
-To do that, switch to the `docs` directory in the repository:
+`cd` to the `docs` directory in the repository:
 
-```
+```zsh
 cd docs
 ```
 
-Download the documentation tarballs from the daml and canton repos:
+### Download docs
 
-```
+Run the following script to download the documentation tarballs from the Daml and Canton repos:
+
+```zsh
 ./download.sh
 ```
 
-## Preview
+### Live preview
 
-For local development, use the `live-preview.sh` script. The files for the
-unified documentation will be in the `docs/workdir/build` folder; changes to
-those files should be immediately reflected.
+Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits should be immediately reflected.
```

Comment:
This seems a but ambiguous to me: you should definitely not edit the HTML files. Perhaps:
```suggestion
Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits to `docs/index/` files should be immediately reflected.
```